Backpack-SwiftUI/Calendar/Classes/Core/CalendarTypeContainerFactory.swift

```diff
@@ -19,9 +19,12 @@
 import SwiftUI
 
 public struct CalendarAccessibilityConfiguration {
-    public let singleSelection: SingleDayAccessibilityProvider
-    public let rangeSelection: RangeDayAccessibilityProvider
-    public init(singleSelection: SingleDayAccessibilityProvider, rangeSelection: RangeDayAccessibilityProvider) {
+    public let singleSelection: SingleDayAccessibilityProvider?
+    public let rangeSelection: RangeDayAccessibilityProvider?
```

Comment:
what'd be the reason for instantiating a configuiration with both values nil? should this be an enum instead?